gradle/libs.versions.toml

```
@@ -15,8 +15,8 @@ androidx-preference = "1.2.1"
 kotlin = "2.1.10-firework.33"
 
 compose-android = "1.7.6"
```

Please keep android and multiplatform versions in sync

```suggestion
compose-android = "1.8.0-beta01"
```

---

Thanks for catching that!